### ISSUE TITLE
Discontinued?

### ISSUE THREAD

**Issue by NetBUG** [Nanoseb/ncTelegram#50]
GIven tg-cli seems to be broken for now without any clear plans to be fixed, does it mean ncTelegram is toast as well?

**Comment by Nanoseb** [comment]
Yes, it unfortunately does. Though, even with tg-cli working, I am not sure what the state of ncTelegram is at the moment as I haven't maintained it in years now.
